### PR TITLE
add postgres94 and mongo26 versions

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@hhorak @adellape ptal.  Note that we'll be adding `registry.access.redhat.com/rhscl/postgresql-94-rhel7` around the time 3.1 ships but it's not available yet.. not sure if you'd rather add it now and escape it vs putting it in later.

File: using_images/db_images/mongodb.adoc [hunk 1]
```diff
@@ -205,9 +209,9 @@ In both cases, if the password was changed correctly, you should see this output
 
 ====
 ----
-MongoDB shell version: 2.4.9
+MongoDB shell version: 2.69
```
Comment (hhorak) [inline using_images/db_images/mongodb.adoc]: missing '.'

**Comment by hhorak** [comment]
Sounds good to me. I think we can add it now already, the name shouldn't change.

File: using_images/db_images/mongodb.adoc (hunk 1 above)
Comment (bparees) [inline using_images/db_images/mongodb.adoc]: pretty sure i pasted that from the actual image output.  i'll double check.

Comment (bparees) [inline using_images/db_images/mongodb.adoc]: nope, i screwed up. fixed, thanks for the catch.

File: using_images/db_images/mongodb.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ provided via configuration.
 
 == Versions
 Currently, OpenShift provides version
-https://github.com/openshift/mongodb/tree/master/2.4[2.4] of MongoDB.
+https://github.com/openshift/mongodb/tree/master/2.4[2.4] and https://github.com/openshift/mongodb/tree/master/2.6[2.6]of MongoDB.
```
Comment (adellape) [inline using_images/db_images/mongodb.adoc]: Missing space after `[2.6]`.

Comment (bparees) [inline using_images/db_images/mongodb.adoc]: fixed

**Pull request merged**